frontend/src/app/app-routing.module.ts

```diff
@@ -4,7 +4,7 @@ import { RouterModule, Routes } from '@angular/router';
 const routes: Routes = [
     {
         path: '',
-        loadChildren: () => import('./modules/landing/landing.module').then((m) => m.LandingModule),
+        loadChildren: () => import('@modules/authentication/authentication.module').then((m) => m.AuthenticationModule),
     },
     {
         path: 'main',
```

Comment:
are you sure that authentication pages should have root path ('') and all other paths (**)? Maybe auth should appear only when there is a 401 status code or 'login/register' route. So in this case mark this code as a temporary solution

Reply:
Yes, I need login component to addit thid by default, so firstly I need to finish this one